### ISSUE TITLE
Where do I need to put this "Zoom to cover width" feature?

### ISSUE THREAD

**Issue by pekka** [fancyapps/fancybox#1740]
I want to change the behaviour of a copy of fancybox so that all photos are zoomed so they cover the full screen width, are centered, and you can use your mouse to pan up and down to see the rest of the image.

I've managed to implement the feature itself in a `scaleToCoverX()` function ([here](https://gist.github.com/pekka/7314de3fbf7a2beeacec00d763e3c9e7), very rough and not tested a lot yet).  

This works. However, no matter where I try to fire the call - say in the `afterShow` event - the image appears for a short moment in its original "scale to fit" size and is enlarged then. That is very annoying.

I'm having a hard time finding where I need to put this to alter Fancybox's original "scale to fit" calculation. It doesn't seem to be the `scaleToFit()` function, I tried to override that one but to no effect.

Can you give me a hint where I'd need to put this for it to show up organically?

I'm happy to provide a pull request once I'm done, if you think this feature is worth having in Fancybox.

**Comment by mjcampagna** [comment]
Rather than modifying the existing code and zoom function, you'd probably have an easier time creating a custom button triggering a new, separate zoom function. This would also future-proof you against updates.

**Comment by pekka** [comment]
That's true! But I can already do what I need with a button. I need it to be the automatic, default behaviour though. Going to be more work to keep current with future updates, for sure.
